app/views/chats/_chat_card.html.erb

```diff
@@ -22,6 +22,9 @@
             <% if current_account && current_account.chats.include?(chat) %>
                 <p>Visited</p>
             <% end %>
+            <div class="logs-link justify-centre">
+                <p>Logs</p><%= link_to chat.url+"/log", chat %>
```

Comment:
Incorrect usage of link_to. It should be something like this.

```erb
<%= link_to "Log", log_chat_path(chat), target: "_blank" %>
```

I also think this might be an inappropriate place to add the link. Up near the title of the card where the chat name is shown would be preferable.

Reply:
ah, thanks! i'm not super familiar with ruby/rails, so that's my bad

the latest commit should put it below the title, but above any group chat descriptions, and still centered. (this is, again, unfortunately untested as it's too late for me to do the whole setup)